model/common_strings.smithy

```diff
@@ -247,6 +247,11 @@ string PathTaskId
 @documentation("The name of the template.")
 string PathTemplateName
 
+@xDataType("array")
```

Comment:
@nhtruong,  
Do you think this is required here? `@xDataType("array") `

Reply:
Yes. All comma separated values in `path parameters` are actually array

Reply:
Btw, since it's an array, the name should be `PathStats`

Reply:
Looks like the values of PathStats are limited to certain enum values. So this will need `xEnumOptions` as well.